models/action_test.go

```diff
@@ -173,7 +173,7 @@ func TestPushCommits_AvatarLink(t *testing.T) {
 	pushCommits.Len = len(pushCommits.Commits)
 
 	assert.Equal(t,
-		"https://secure.gravatar.com/avatar/ab53a2911ddf9b4817ac01ddcd3d975f?d=identicon",
+		"/user/avatar/user2/-1",
```

Comment:
```suggestion
		"/suburl//user/avatar/user2/-1",
```

Reply:
Fixed also removing the duplicated slash